godot-core/src/registry/method.rs

```diff
@@ -10,12 +10,12 @@ use sys::interface_fn;
 
 use crate::builtin::{StringName, Variant};
 use crate::global::MethodFlags;
-use crate::meta::{ClassName, PropertyInfo, VarcallSignatureTuple};
+use crate::meta::{ClassName, GodotConvert, GodotType, ParamTuple, PropertyInfo};
 use crate::obj::GodotClass;
 
 /// Info relating to an argument or return type in a method.
 pub struct MethodParamOrReturnInfo {
-    info: PropertyInfo,
+    pub info: PropertyInfo,
```

Comment:
Making this `pub` is intended? 

See also main comment about `MethodParamOrReturnInfo`, which is currently _not_ public. If it becomes public, we should probably think about the encapsulation.